frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpanded.tsx

```diff
@@ -211,19 +211,27 @@ const CallToActionSection = ({
   const platformImage =
     PLATFORMS[resource?.platform?.code as PlatformEnum]?.image
 
-  const { resource_type: type, platform } = resource!
+  const { platform } = resource!
 
-  const cta =
-    type === ResourceTypeEnum.Podcast ||
-    type === ResourceTypeEnum.PodcastEpisode
-      ? "Listen to Podcast"
-      : `Take ${getReadableResourceType(type)}`
+  const getCallToActionText = (resource: LearningResource): string => {
+    if (resource?.platform?.code === PlatformEnum.Ocw) {
+      return "Access Course Materials"
+    } else if (
+      resource?.resource_type === ResourceTypeEnum.Podcast ||
+      resource?.resource_type === ResourceTypeEnum.PodcastEpisode
+    ) {
+      return "Listen to Podcast"
+    }
+    return "Learn More"
+  }
 
+  const cta = getCallToActionText(resource)
   return (
     <CallToAction>
       <StyledLink
```

Comment:
@shanbady Could you change the StyledLink width from 220px to 224px? Then "Access Course Materials" will be one line.

Sorry for the confusion on this. We [changed](https://github.com/mitodl/hq/issues/4784) the text-to-icon spacing a while back and never updated Figma, so what Figma thinks fits within 220px and what actually fits within 220px are slightly different at the moment.

cc @steven-hatch @mbilalmughal